extensions/interactions/EndExploration/EndExploration.html

```diff
@@ -17,8 +17,26 @@
       </md-card>
     </div>
   </div>
+  <div ng-if="recommendedExplorationIds.length > 0 && collectionId">
+    Continue the collection:
+    <div layout="row" layout-wrap>
+      <md-card class="oppia-end-exploration-recommendation-card" ng-repeat="expId in recommendedExplorationIds track by $index">
+        <a ng-href="/explore/<[expId]>?collection_id=<[collectionId]>" target="<[isIframed ? '_blank' : '_self']>" ng-if="recommendedExplorationSummaries[$index]">
+          <img class="oppia-end-exploration-recommendation-image" ng-src="<[recommendedExplorationSummaries[$index].thumbnail_image_url]>">
+          <div class="oppia-end-exploration-recommendation-title">
+            <[recommendedExplorationSummaries[$index].title]>
+          </div>
+        </a>
+      </md-card>
+    </div>
+  </div>
 
   <div ng-if="!isIframed">
+    <a href="/collection/<[collectionId]>" ng-if="collectionId && !isInEditorPreviewMode">
```

Comment:
ng-href

Also, may be worth moving the common !isInEditorPreviewMode check to a common div around the two links.